selection-to-clipboard/manifest.json

```diff
@@ -9,14 +9,8 @@
         "48": "icons/clipboard-48.png"
     },
 
-    "applications": {
-        "gecko": {
-            "strict_min_version": "51.0a1"
```

Comment:
No. The original PR asked for the "clipboardWrite" permission, that was added in 51. But the example doesn't actually need that permission, because the copy is done in an event handler for a user action. So I asked the contributor to remove the permission, so as not to suggest to people that it is in fact needed.

The example works fine in Firefox 50.